website/docs/configuration/providers.html.md

```diff
@@ -119,32 +81,42 @@ select which one to use on a per-resource or per-module basis. The primary
 reason for this is to support multiple regions for a cloud platform; other
 examples include targeting multiple Docker hosts, multiple Consul hosts, etc.
 
-To include multiple configurations for a given provider, include multiple
-`provider` blocks with the same provider name, but set the `alias` meta-argument
-to an alias name to use for each additional configuration. For example:
+To create multiple configurations for a given provider, include multiple
+`provider` blocks with the same provider name. For each additional non-default
+configuration, use the `alias` meta-argument to provide an extra name segment.
+For example:
 
 ```hcl
-# The default provider configuration
+# The default provider configuration; resources that begin with "aws_" will use
+# it as the default, and it can be referenced as "aws".
```

Comment:
Yeah, that's probably a good idea. I was using quotes because this isn't a context where backticks have a formatting effect, but I guess Markdown is now so common that most coders understand them as "this is a verbatim computer word, not a normal quote", so we might as well lean on that!

Reply:
Oh yes, good point! I was reading this as Markdown source code rather than rendered and so I didn't quite notice that this particular "paragraph" was part of a code example rather than a separate paragraph.

I don't find myself _so_ convinced that all/most readers would understand what backticks presented literally would mean here, though you may be right. I have an alternative proposal if not, which I will admit reads awkwardly as an English sentence but I think makes it clearer what part is intended to be taken literally:

```
# The default provider configuration; resources that begin with "aws_" will use
# it as the default, and it can be referenced as: provider = aws
```

I don't love it by any means. Perhaps this is an ugly enough sentence that the quotes are better in spite of the ambiguity! :grimacing: I'll leave the final call up to you, of course.